codespell_lib/data/dictionary.txt

```diff
@@ -35055,7 +35124,7 @@ unsiged->unsigned
 unsigend->unsigned
 unsignd->unsigned
 unsignificant->insignificant
-unsing->using
+unsing->using, unsign,
```

Comment:
```text
unsing->using, unsigned,
```
`unsign` is not a valid word in various dictionaries (aspell, hunspell, ispell, aoo-mozilla-en-dict, browser).
Please suggest `unsigned`. If you type `unsign integer` in Google search box before hitting ENTER key,
you will notice Google suggest you with `unsigned integer` for your search